app/src/main/java/com/chaubacho/calculator/MainActivity.kt

```diff
@@ -0,0 +1,103 @@
+package com.chaubacho.calculator
+
+import android.os.Bundle
+import android.util.Log
+import android.view.View
+import android.widget.Button
+import androidx.appcompat.app.AppCompatActivity
+import kotlinx.android.synthetic.main.activity_main.*
```

Comment:
Kotlin Android Extensions is deprecated, which means that using Kotlin synthetics for view binding is no longer supported. If your app uses Kotlin synthetics for view binding, use this guide to migrate to Jetpack view binding.
https://developer.android.com/topic/libraries/view-binding/migration

Reply:
dạ vâng để e chuyển sang view binding ạ